Gracefully handle Slack duplicate installation attempt

If you attempt to re-install into a slack workspace where EPYC is already installed, it should gracefully deal with this by overwriting the installation data (ie store the updated oauth token).  Instead it fails with a mongo uniqueness error:

```
8|epyc  | [ERROR]  OAuth:InstallProvider:0 MongoError: E11000 duplicate key error collection: 5f83803e0ca4614ab1c99ff9_Epyc.SlackToken index: _id_ dup key: { _id: "<team ID>" }
8|epyc  |     at Function.create (/home/www/epyc/node_modules/mongodb/lib/core/error.js:57:12)
8|epyc  |     at toError (/home/www/epyc/node_modules/mongodb/lib/utils.js:123:22)
8|epyc  |     at /home/www/epyc/node_modules/mongodb/lib/operations/common_functions.js:379:39
8|epyc  |     at handler (/home/www/epyc/node_modules/mongodb/lib/core/topologies/replset.js:1204:22)
8|epyc  |     at /home/www/epyc/node_modules/mongodb/lib/core/connection/pool.js:405:18
8|epyc  |     at processTicksAndRejections (node:internal/process/task_queues:75:11) {
8|epyc  |   driver: true,
8|epyc  |   index: 0,
8|epyc  |   code: 11000,
8|epyc  |   keyPattern: { _id: 1 },
8|epyc  |   keyValue: { _id: '<team ID>' }
8|epyc  | }
```

--- a/server/Db.ts
+++ b/server/Db.ts
@@ -401,7 +401,7 @@ export class Db implements IDb {
         slackToken.teamId = teamId;
         slackToken.installation = installation;
 
-        await this.slackToken.replaceOne({ id: teamId }, deflate(slackToken), { upsert: true });
+        await this.slackToken.replaceOne({ _id: teamId }, deflate(slackToken), { upsert: true });
     }
 
     private async resolveLinkedChannels(channel: ChannelModel): Promise<ChannelModel[]> {

